api/src/backend/api/v1/views.py

```diff
@@ -1349,15 +1322,8 @@ class FindingViewSet(BaseRLSViewSet):
         "inserted_at",
         "updated_at",
     ]
-    required_permissions = []
-    permission_classes = BaseRLSViewSet.permission_classes + [HasPermissions]
-
-    def initial(self, request, *args, **kwargs):
-        """
-        Sets required_permissions before permissions are checked.
-        """
-        self.required_permissions = ResourceViewSet.required_permissions
-        super().initial(request, *args, **kwargs)
+    # RBAC required permissions
+    required_permissions = [Permissions.MANAGE_PROVIDERS, Permissions.MANAGE_SCANS]
```

Comment:
Why `Permissions.MANAGE_SCANS`?

This is a readonly viewset. Same with overviews. Maybe I'm missing something.

Reply:
Maybe we don't have the same idea behind the permission, I think we should have a common place where this is defined, we can discuss it but in principle in that endpoint you can see findings and for me the findings are related to scans.